core/src/main/java/org/elasticsearch/index/engine/SegmentsStats.java

```diff
@@ -292,6 +306,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.byteSizeField(Fields.INDEX_WRITER_MEMORY_IN_BYTES, Fields.INDEX_WRITER_MEMORY, indexWriterMemoryInBytes);
         builder.byteSizeField(Fields.VERSION_MAP_MEMORY_IN_BYTES, Fields.VERSION_MAP_MEMORY, versionMapMemoryInBytes);
         builder.byteSizeField(Fields.FIXED_BIT_SET_MEMORY_IN_BYTES, Fields.FIXED_BIT_SET, bitsetMemoryInBytes);
+        builder.field(Fields.MAX_UNSAFE_AUTO_ID_TIMESTAMP, maxUnsafeAutoIdTimestamp);
```

Comment:
I don't think we should do this - it's an impl detail and we might go and use something else ie. a sequence id later that will mess things up?